yarn.lock

```diff
@@ -13444,10 +13444,9 @@ __metadata:
   version: 0.0.0-use.local
   resolution: "root-workspace-0b6124@workspace:."
   dependencies:
-    "@babel/core": ^7.13.10
-    "@polkadot/dev": ^0.62.5
-    "@types/jest": ^26.0.20
-    bs58: ^4.0.1
+    "@babel/core": ^7.13.14
+    "@polkadot/dev": ^0.62.7
+    "@types/jest": ^26.0.22
```

Comment:
Let's not change this file @AmazingBeerBelly. Those are incremental changes. I think we should leave them as-is.